sycl/test-e2e/InvokeSimd/Spec/simd_mask_merge.cpp

```diff
@@ -100,5 +100,5 @@ int main() {
   }
 
   std::cout << (err_cnt > 0 ? "FAILED\n" : "Passed\n");
-  return err_cnt == 0;
+  return err_cnt != 0;
```

Comment:
I think this would be better:
```suggestion
  assert(err_cnt == 0);
  return 0;
```

Reply:
done in latest commit, thx

Reply:
I actually think the original version was better. 
If somehow (by additional options maybe) the assert are turned off, then the test would return pass.

Reply:
i can change it back in another pr if @aelovikov-intel is okay with it

Reply:
We have so many tests that rely on the assert (https://github.com/search?q=repo%3Aintel%2Fllvm+assert+path%3Asycl%2Ftest-e2e&type=code) that I think this argument is moot.